src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -626,7 +627,18 @@ int goto_analyzer_parse_optionst::perform_analysis(const optionst &options)
 
     if(options.get_bool_option("show"))
     {
-      static_show_domain(goto_model, *analyzer, options, out);
+      if(
```

Comment:
I think I would prefer a new option for output "text-on-source" or something like that and have ```show_on_source``` just called for that.  That way we maintain backwards compatibility.

Reply:
Ok, it's now --show-on-source.